next-tavla/src/Admin/scenarios/Edit/index.tsx

```diff
@@ -28,10 +38,23 @@ function Edit({
                 <TavlaButton
                     onClick={() => {
                         setBoardSettings(documentId, settings)
+                        setShowLink(!showLink)
                     }}
                 >
                     Lagre instillinger
                 </TavlaButton>
+                {showLink && (
+                    <div>
+                        <CopyText text={linkURL} />
+                        <PrimaryButton
+                            onClick={() => {
+                                handleClick()
+                            }}
+                        >
+                            Se avgangstavla
+                        </PrimaryButton>
```

Comment:
Siden denne knappen bare navigerer kan vi bruke Nextjs sin [Link-komponent](https://nextjs.org/docs/pages/building-your-application/routing/linking-and-navigating) her og style den istedet. Det blir lettere for bl.a. skjermlesere, og vi får litt ekstra på kjøpet med prefetching